Source/ui_qt/DebugSupport/DebugView.cpp

```diff
@@ -2,7 +2,7 @@
 #include "BiosDebugInfoProvider.h"
 #include <QVBoxLayout>
 
-CDebugView::CDebugView(QMdiArea* parent, CVirtualMachine& virtualMachine, CMIPS* ctx,
+CDebugView::CDebugView(QWidget* parent, QMdiArea* midArea, CVirtualMachine& virtualMachine, CMIPS* ctx,
```

Comment:
I guess this is a mistake, but the variable is named `midArea`. Shouldn't it be `mdiArea`?

Reply:
indeed, changes pushed.